README.md

```diff
@@ -99,127 +99,142 @@ val config: Try[YourConfClass] = loadConfig[YourConfClass]
-> import com.typesafe.config.ConfigFactory.parseString
-> sealed trait MyAdt
-> case class AdtA(a: String) extends MyAdt
-> case class AdtB(b: Int) extends MyAdt
-> case class MyClass(int: Int, adt: MyAdt, list: List[Double], map: Map[String, String], option: Option[String])
-> val conf = parseString("""{ "int": 1, "adt": { "type": "adtb", "b": 1 }, "list":["1", "20%"], "map": { "key": "value" } }""")
-> loadConfig[MyClass](conf)
-res0: util.Try[MyClass] = Success(MyClass(1,AdtB(1),List(1.0, 0.2),Map(key -> value),None))
+import com.typesafe.config.ConfigFactory.parseString
+
+sealed trait MyAdt
+case class AdtA(a: String) extends MyAdt
+case class AdtB(b: Int) extends MyAdt
+case class MyClass(int: Int, adt: MyAdt, list: List[Double], map: Map[String, String], option: Option[String])
+
+val conf = parseString("""{ "int": 1, "adt": { "type": "adtb", "b": 1 }, "list": ["1", "20%"], "map": { "key": "value" } }""")
+
+loadConfig[MyClass](conf)
+// returns Success(MyClass(1, AdtB(1), List(1.0, 0.2), Map("key" -> "value"), None))
 ```
 
 
 ## Configurable converters
 
-For some types, pureconfig cannot automatically derive a converter because there are multiple ways to convert a configuration
+For some types, PureConfig cannot automatically derive a converter because there are multiple ways to convert a configuration
 value to them. For instance, for [`LocalDate`](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html)
-pureconfig cannot derive a converter because there are multiple [`DateTimeFormatter`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)s
-that can be used to convert a String into a `LocalDate`. Examples of different format are `yyyy-mm-dd`, e.g. `"2016-01-01"`,
-and `yyyymmdd`, e.g. `"20160101"`. For those types, pureconfig provides a way to create converters from the necessary parameters. These methods can be found under
+PureConfig cannot derive a converter because there are multiple [`DateTimeFormatter`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)s
+that can be used to convert a String into a `LocalDate`. Examples of different formats are `yyyy-mm-dd`, e.g. `"2016-01-01"`,
+and `yyyymmdd`, e.g. `"20160101"`. For those types, PureConfig provides a way to create converters from the necessary parameters. These methods can be found under
 the package `pureconfig.configurable`. Once the output of a `pureconfig.configurable` method for a certain type is in scope,
-pureconfig can start using that configured converter:
+PureConfig can start using that configured converter:
 
 ```scala
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
 import pureconfig.configurable._
 
 case class Conf(date: LocalDate)
 
 implicit val localDateInstance = localDateConfigConvert(DateTimeFormatter.ISO_DATE)
 
-val conf = ConfigFactory.parseString(s"""{date:"2011-12-03"}""")
+val conf = parseString(s"""{ date: "2011-12-03" }""")
```

Comment:
I think we should import the methods that we use, otherwise what is `parseString`? I'm thinking in particular about developers that don't know Typesafe config.